packages/cli/src/handlers/validate.ts

```diff
@@ -63,26 +63,33 @@ export const validateHandler = async (options: ValidateHandlerOptions) => {
 
     const config = await getConfig();
 
-    const projectUuid = options.project || config.context?.previewProject;
+    const projectUuid =
+        options.project ||
+        config.context?.previewProject ||
+        config.context?.project;
 
     if (projectUuid === undefined) {
         throw new ParameterError(
             `No project specified, select a project to validate using ${styles.bold(
                 `--project <projectUuid>`,
             )} or create a preview environment using ${styles.bold(
                 `lightdash start-preview`,
+            )} or configure your default project using ${styles.bold(
+                `lightdash config set-project`,
             )}`,
         );
     }
 
     if (options.project) {
         console.error(`Validating project ${projectUuid}\n`);
-    } else {
+    } else if (config.context?.previewProject) {
         console.error(
             `Validating preview project ${styles.bold(
                 config.context?.previewName,
-            )} with projectUuid ${config.context?.previewProject}\n`,
+            )} with projectUuid ${projectUuid}\n`,
```

Comment:
Can we change this to the project name? 

So like: `Validating preview project **Humble Bumblebee**` (where the project name is bold)

Or `Validating project **Lightdash Analytics**`


^I know that projects can have the same name, but it's really unlikely for them to be named the same and I feel like the name is more intuitive/helpful than the UUID

Reply:
at the moment we don't store the project name, but I can add it , let me change this